config/locales/en.yml

```diff
@@ -40,7 +40,7 @@ en:
     edit:
       header: 'Edit your Team'
       legend: 'New Team Information'
-      note: 'All fields are required.'
+      looking_for_members_tooltip: 'When the looking for members checkbox is checked, your team will be displayed as a joinable team for those on the join teams page'
```

Comment:
Since this is displayed right next to the looking for members checkbox, this should be able to just say:

> When this box is checked, your team will be displayed for users searching to join a team. When unchecked you will still be able to invite new users to your team by email.